users/vieting/experiments/switchboard/ctc/feat/data.py

```diff
@@ -74,10 +80,21 @@ def get_corpus_data_inputs_oggzip(
     meta_args = {"data_map": {"classes": ("hdf", "data"), "data": ("ogg", "data")}}
     if context_window is not None:
         meta_args["context_window"] = context_window
+    if preprocessing:
+        audio = {
+            "features": "raw",
+            "peak_normalization": True,
+            "pre_process": CodeWrapper("audio_perturb_runner.run")
+        }
+    else:
+        audio = {
+            "features": "raw",
+            "peak_normalization": True
+        }
```

Comment:
You can also set this by default and then `if pre_process is not None: audio["pre_process"] = pre_process`.

Reply:
Done